### DISCUSSION TITLE
missing releases 6.4.2, 6.5.0 and 6.5.1

### DISCUSSION THREAD

**Discussion by omissis** [stoplightio/spectral#2252] [Q&A] [answered]
Hi everyone,
  I was updating some docker and [asdf](https://github.com/asdf-vm/asdf) dependencies when I noticed releases 6.4.2, 6.5.0 and 6.5.1 are not available on github(the tag is there but the release is not), while they are present on npmjs.com.
I was wondering if this is a wanted behavior, as the asdf plugin still sees 6.4.1 (the latest published on github) as the latest, while there have been 3 more versions since then.
Thanks!

**Comment by P0lip** [comment, marked as answer]
I've just released them. Apologies for a delay.

**Comment by omissis** [comment]
thanks a lot!
